config/constants/settings/definition.rb

```diff
@@ -728,6 +728,16 @@ class Definition
         },
         writable: false
       },
+      remote_storage_upload_host: {
+        format: :string,
+        default: nil,
+        writable: false
+      },
+      remote_storage_download_host: {
+        format: :string,
+        default: nil,
+        writable: false
```

Comment:
Please add descriptions to those settings

Reply:
@ulferts I've added the descriptions.